src/functions/commonFunc.js

```diff
@@ -0,0 +1,24 @@
+export const calculatePercentage = (count, price) => {
+  let calculatedPrice = 0;
+  if (typeof count === 'string') Number(count);
+  if (count >= 1 && count < 5) {
+    //5개 미만 미할인
+    calculatedPrice = price;
+  } else if (count >= 5 && count < 10) {
+    //5개 이상 5% 할인
+    calculatedPrice = Math.floor((price * 0.95) / 10) * 10;
+  } else if (count >= 10 && count < 30) {
+    //10개 이상 8% 할인
+    calculatedPrice = Math.floor((price * 0.92) / 10) * 10;
+  } else if (count >= 30 && count < 50) {
+    //30개 이상 10% 할인
+    calculatedPrice = Math.floor((price * 0.9) / 10) * 10;
+  } else if (count >= 50) {
+    //50개 이상 13% 할인
+    calculatedPrice = Math.floor((price * 0.87) / 10) * 10;
+  } else {
+    calculatedPrice = price;
+  }
```

Comment:
```suggestion
  const discountStandard = [50, 30, 10, 5];
  const discountPercent = [0.87, 0.9, 0.92, 0.95];

  const StandardIndex = discountStandard.indexOf(
    discountStandard.filter(number => count >= number)[0]
  );

  if (StandardIndex === -1) {
    calculatedPrice = price;
  } else {
    calculatedPrice =
      Math.floor((price * discountPercent[StandardIndex]) / 10) * 10;
  }
```

조건문이 연속적으로 되어있고 반복되는 로직이 있어서 이 부분 위와같은 방식으로 줄일 수 있겠습니다!